lib/sidekiq_jobs.rb

```diff
@@ -34,10 +35,82 @@
 end
 
 module Jobs
+  # make functions available as Jobs::whatever
```

Comment:
I tried to move the responsibility for serializing/deserializing things to JSON (or json-able types) here rather than any of the actual jobs needing to know anything about that.